frontend/src/metabase/admin/databases/database.js

```diff
@@ -129,7 +116,7 @@ export const initializeDatabase = function (databaseId) {
         auto_run_queries: true,
         engine: getDefaultEngineKey(engines),
         details: {},
-        created: false,
```

Comment:
value wasn't used anywhere, not having the created_at time breaks `Database` type